### DISCUSSION TITLE
Go Index

### DISCUSSION THREAD

**Discussion by craigrj** [dtcenter/METplus#1373] [Configuration] [answered]
Go Index takes two model names, the model and the reference.  If I call go_index with model1, model2 and get a value of <1 for the index, which model is better.  The documentation doesn't specify from what I can tell.  Thanks

**Comment by georgemccabe** [comment, marked as answer]
Hi @craigrj,

The "latest" version of the MET documentation, which corresponds to the latest stable release, does not clearly describe how to interpret the Go Index (or SS Index) output values. However, there have been changes to the docs since that release that do contain a good explanation. They can be found in the "develop" version of the MET documentation here: https://met.readthedocs.io/en/develop/Users_Guide/stat-analysis.html#skill-score-index

Please let us know if this is still unclear and we can update it to be more descriptive.
